libs/stdlib/src/bit_shift_functions.rs

```diff
@@ -1,61 +1,5 @@
 //! Defines shift operations
 
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift left operation on bytes
-pub fn SHL__BYTE(input: u8, n: u32) -> u8 {
-    input << n
-}
-
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift left operation on word
-pub fn SHL__WORD(input: u16, n: u32) -> u16 {
-    input << n
-}
-
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift left operation on dword
-pub fn SHL__DWORD(input: u32, n: u32) -> u32 {
-    input << n
-}
-
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift left operation on lword
-pub fn SHL__LWORD(input: u64, n: u32) -> u64 {
-    input << n
-}
-
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift right operation on bytes
-pub fn SHR__BYTE(input: u8, n: u32) -> u8 {
-    input >> n
-}
-
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift right operation on word
-pub fn SHR__WORD(input: u16, n: u32) -> u16 {
-    input >> n
-}
-
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift right operation on dword
-pub fn SHR__DWORD(input: u32, n: u32) -> u32 {
-    input >> n
-}
-
-#[allow(non_snake_case)]
-#[no_mangle]
-/// Shift right operation on lword
-pub fn SHR__LWORD(input: u64, n: u32) -> u64 {
-    input >> n
-}
-
```

Comment:
@ghaith why the migration from SH{L,R} as a standard library function to a built-in one? Shouldn't we try to keep the built-ins as small as possible?